### DISCUSSION TITLE
LongBench-Chat评测Qwen2.5-72B-Instruct指标不一致问题

### DISCUSSION THREAD

**Discussion by ZayIsAllYouNeed** [QwenLM/Qwen3#1148] [Q&A]
尊敬的作者、同行，您好，我用LongBench-Chat的官方代码（https://github.com/THUDM/LongAlign/tree/main/LongBench_Chat）评测了Qwen2.5-72B-Instruct的128K上下文指标，
（将代码中的transformers推理更改为vllm推理，Qwen的config.json加上YaRN外推128K配置，其他参数均未变），
评估2次结果分别为8.42、8.44，没有达到技术报告上的8.72，请问评测过程有什么需要特别注意的地方吗？
我需要做什么以复现8.72的结果呢？

**Comment by jklj077** [comment]
cc: @hzhwcmhf

**Comment by xs1997zju** [comment]
+1 无法复现技术报告中的LongBenchChat指标
